presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -434,6 +434,31 @@ public void testPartitionPredicateDisallowed() throws Exception
         assertUpdate(session, "DROP TABLE partition_test12");
     }
 
+    @Test
+    public void testNoPartitionFilterCheckOnAnalyze()
+    {
+        Session session = Session.builder(getQueryRunner().getDefaultSession())
+                .setIdentity(Identity.forUser("hive")
+                        .withRole("hive", new SelectedRole(ROLE, Optional.of("admin")))
+                        .build())
+                .setCatalogSessionProperty("hive", "query_partition_filter_required", "true")
+                .build();
+
+        assertUpdate(
+                session,
+                "create table partition_test14(\n"
+                        + "id integer,\n"
+                        + "a varchar,\n"
+                        + "b varchar,\n"
```

Comment:
replace `\n` with ` ` for readability

(you can also put all columns on one line)